packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -8,12 +8,19 @@ import { processEnvVariables } from "../../util/processEnvVariables.js";
 import { nodePolyfills } from "vite-plugin-node-polyfills";
 import pc from "picocolors";
 import { addResponseHeadersToConfigYaml } from "../../util/editConfigYaml.js";
+// @ts-expect-error due to type any
+import scopeTailwind from "vite-plugin-scope-tailwind";
 import SourceFileParser, {
   createTsMorphProject,
 } from "../../common/src/parsers/sourceFileParser.js";
+import { logWarning } from "../../util/logError.js";
+import postcss from "postcss";
+import nested from "postcss-nested";
 
 const wrappedCode = (moduleName: string, containerName: string): string => {
-  return `\nconst moduleContainerForBuildUseOnly = document.getElementById('${containerName}');
+  return `
+// The following code is added during build and removed after build is completed.
```

Comment:
You should be able to use rollup's `outro` option to handle this instead of modifying the original file.

Reply:
You can use `intro` for the react imports.